### PR TITLE
LB loading animation added

### PR DESCRIPTION
**Pull request by 07jasjeet**
Experimental animation but quite stable with very less recompositions.

### CODE REVIEW

File: app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+ * @param rotateDuration Duration of rotate animation.
+ */
+@Preview
+@Composable
+fun LBLoadingAnimation(
+    logoSize: Dp = 60.dp,
+    middleGap: Dp = logoSize / 20,
+    sideOffset: Dp = logoSize / 4,
+    backgroundColor: Color = Color.Transparent,
+    lb_orange: Color = Color(0xFFEA743B),
```
Comment (akshaaatt) [inline app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt:43]: We should specify generic names for these parameters. Also, this composable could just be a LoadingAnimation.

**Review by akshaaatt — APPROVED** [review]
LGTM! Awesome work 💯

File: app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt (hunk 1 above)
Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt:43]: These are actually supposed to be constant as is and these are the best generic names my brain could cook up 😭

Comment (akshaaatt) [reply app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt:43]: leftColor and rightColor could be the names @07jasjeet . The idea is to generalize components with dynamic animation options/colors, etc.

Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt:43]: oh the color params I see.

Comment (07jasjeet) [reply app/src/main/java/org/listenbrainz/android/ui/components/LBLoadingAnimation.kt]: Done

**Review by akshaaatt — APPROVED** [review]

**Pull request merged**